.ci/docker/manywheel/Dockerfile_2_28_aarch64

```diff
@@ -38,6 +38,12 @@ RUN yum install -y \
   sudo \
   gcc-toolset-${GCCTOOLSET_VERSION}-toolchain
 
+RUN cd ~/ && \
+  wget https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux-aarch64.zip && \
```

Comment:
@tinglvv Can we update https://github.com/pytorch/pytorch/blob/main/.ci/docker/common/install_ninja.sh to just support arm as well?

Reply:
```bash
#!/bin/bash

set -ex

[ -n "$NINJA_VERSION" ]

arch=$(uname -m)
if [ "$arch" == "aarch64" ]; then
    url="https://github.com/ninja-build/ninja/releases/download/v${NINJA_VERSION}/ninja-linux-arm64.zip"
else
    url="https://github.com/ninja-build/ninja/releases/download/v${NINJA_VERSION}/ninja-linux.zip"
fi

pushd /tmp
wget --no-verbose --output-document=ninja-linux.zip "$url"
unzip ninja-linux.zip -d /usr/local/bin
rm -f ninja-linux.zip
popd
```

Reply:
note: you can just add the NINJA_VERSION arg to the docker to match all the dockers as well. :)